frontend/src/routes/Credentials/CreateCredentialsCatalog.tsx

```diff
@@ -33,19 +32,31 @@ export function CreateCredentialsCatalog() {
 
   const cards = useMemo(() => {
     const newCards: ICatalogCard[] = [
-      ...orderedProviders
-        // does not display AWS s3 card
-        .filter(([id]) => id !== Provider.awss3)
```

Comment:
Why did you remove this filtering?

Reply:
I assumed since the S3 credential card has been moved to a secondary path behind AWS, it was no longer needed on the main catalog page.  Can revert

Reply:
This is the code that prevents it from appearing in the top level.